server/api/controllers/config/environments/environmentsConfigController.js

```diff
@@ -6,12 +6,13 @@ const KEY_NAME = 'EnvironmentName';
 
 let _ = require('lodash');
 let co = require('co');
+let Promise = require('bluebird');
```

Comment:
Promise has been replaced globally - is this to make it explicit within this file?